src/Webserver.cc

```diff
@@ -2,8 +2,6 @@
 // http://www.boost.org/doc/libs/1_55_0/doc/html/boost_asio/example/cpp11/echo/blocking_tcp_echo_server.cpp
 
 #include "request_handler.h"
-#include "echo_handler.h"
```

Comment:
webserver only uses request_handler, so I don't think its needed. maybe its safer to have it? i dunno it runs fine.

Also, I think in static_file_handler there needs to be a check to see if filename = "". if you run localhost:2020/static/ it tries to open it